SUMMARY.md

```diff
@@ -169,6 +169,7 @@
   - [National Registry](libs/clients/national-registry/README.md)
     - [V1](libs/clients/national-registry/v1/README.md)
     - [V2](libs/clients/national-registry/v2/README.md)
+  - [Clients Passports](libs/clients/passports/README.md)
```

Comment:
The title should preferably be overriden using gitbook comments in the library's readme to be:
```suggestion
  - [Passports](libs/clients/passports/README.md)
```

Reply:
Makes sense!

Fixed. 💯